api/pb/v1/oracle.swagger.json

```diff
@@ -0,0 +1,43 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "pb/v1/oracle.proto",
+    "version": "version not set"
```

Comment:
Should this be set somehow? Also, should this file exist? It seems that it wasn't being generated before and I don't understand the purpose of it because it doesn't contain any of the model definitions from oracle.proto

Reply:
the `buf generate` tool chain automatically generates the swagger specs for all the files. I'd have to add some exception to tell it explicitly not to generate it